### ISSUE TITLE
i18n & l10n

### ISSUE THREAD

**Issue by cagcak** [Tipue/Tipue-Search#70]
When user changes the locale of the site, all the text based elements change based on the selection. 
I have two content files as : 

- tipuesearch_content_en.js
- tipuesearch_content_fr.js

When` $.i18n()` library is set to 'fr', I expect also the search result content changes using those two files. I've tried to accomplish this on either static or live modes about a week. I don't want to give up using tipuesearch, 
Any suggestion will be appriciated.
Thanks.

**Comment by jorgesumle** [comment]
I translated it into Spanish for a website, so maybe you can add that language to your website if you find it useful. 

I think that it would be a great idea if you propose a pull request to make it easier for people to internationalize Tipue-Search.

**Comment by cagcak** [comment]
Hey @jorgesumle . I've tried but no luck. Maybe I should remove tipuesearch from my project and move to classic database related approach. But it is so much borring work. I think whole tipuesearch.js must be modified for i18n. 

But if I could use some jquery calls inside of tipuesearch_content.js file, there would be some work around. So, when a user changes the language code, all the results would have been changed.
